test/remapping.jl

```diff
@@ -70,24 +100,15 @@ end
 end
 
 @testset "Finite Volume Remapping" begin
-    quad = Spaces.Quadratures.GL{1}() # FV specification
```

Comment:
Should we call the above test set `Finite Volume <-> Finite Volume Remapping` for clarity?